feat(KNO-7623): add MsTeamsChannelCombobox to docs (#759)
knocklabs/docs: content/in-app-ui/react/teams-kit.mdx

## Using TeamsKit components

> Once you've provided access to the necessary data, you can drop Knock's pre-built components into your React application to immediately set up Microsoft Teams authorization and channel selection for your users for your users. [See the reference](/sdks/react/reference#microsoft-teams-components) for full documentation of these components.

Reviewer: ```suggestion
Once you've provided access to the necessary data, you can drop Knock's pre-built components into your React application to immediately set up Microsoft Teams authorization and channel selection for your users. [See the reference](/sdks/react/reference#microsoft-teams-components) for full documentation of these components.
```
looks like dupe wording here

  }
/>

#### MsTeamsChannelCombobox

<figure>
  <Image
    src="/images/integrations/chat/microsoft-teams/msteamschannelcombobox.png"
    className="rounded-md mx-auto border border-gray-200"
    alt="The MsTeamsChannelCombobox component showing connected channels"
    width={1088}
    height={584}
  />
  <figcaption>
    The MsTeamsChannelCombobox component showing connected channels
  </figcaption>
</figure>

> This combobox contains the list of teams and channels belonging to the connected Microsoft Entra tenant. Users will use this combobox to search and select a channel (or more than one channel) to be notified when an event in your application occurs, for example a comment on a video. They can also use this combobox to deselect a connected channel.

Reviewer: ```suggestion
This combobox contains the list of teams and channels belonging to the connected Microsoft Entra tenant. Users will use this combobox to search and select a channel (or more than one channel) to be notified when your application triggers a workflow with a Teams channel step. They can also use this combobox to deselect a connected channel.
```
Just a suggestion
